projects/igniteui-angular/src/lib/navigation-drawer/navigation-drawer.component.ts

```diff
@@ -805,19 +815,27 @@ export class IgxNavigationDrawerComponent implements
      * @param opacity optional value to apply to the overlay
      */
     private setXSize(x: number, opacity?: string) {
-        // Angular polyfills patches window.requestAnimationFrame, but switch to DomAdapter API (TODO)
-        window.requestAnimationFrame(() => {
-            if (this.hasAnimateWidth) {
-                this.renderer.setElementStyle(this.drawer, 'width', x ? Math.abs(x) + 'px' : '');
-            } else {
-                this.renderer.setElementStyle(this.drawer, 'transform', x ? 'translate3d(' + x + 'px,0,0)' : '');
-                this.renderer.setElementStyle(this.drawer, '-webkit-transform',
-                    x ? 'translate3d(' + x + 'px,0,0)' : '');
-            }
-            if (opacity !== undefined) {
-                this.renderer.setElementStyle(this.overlay, 'opacity', opacity);
-            }
-        });
+        if (this.platformUtil.isPlatformBrowser()) {
```

Comment:
This one shouldn't need a fallback, in fact this is only called during pan interaction, but if you want to make it safe just for future uses - just interrupt the method at start.

Reply:
yeah, will remove that check